apps/src/applab/designElements/textarea.jsx

```diff
@@ -231,7 +231,7 @@ export default {
   onPropertyChange: function (element, name, value) {
     switch (name) {
       case 'value':
```

Comment:
Could we add a comment here to explain why we use the property `value` even though users don't really use `value`?

Reply:
Also - we could consider removing 'value' from the [text area dropdown list](https://github.com/code-dot-org/code-dot-org/blob/6580b67d68b72af16c61a6179b3f6896cc9ca332/apps/src/applab/setPropertyDropdown.js#L396) since users do not use and it's not listed in the `setProperty` documentation.